index.js

```diff
@@ -32,15 +32,51 @@ PLAYERS.forEach(renderPlayer)
 
 
 /***** Deliverable 1 *****/
+let pageHeader = document.querySelector("h1#header")
+
+pageHeader.addEventListener('click', (event) => {
+element = event.target
```

Comment:
This is the right idea, just be careful about declaring variables without `let` or `const` in front. If you don't have `let` or `const` it becomes a global variable, even if it's declared in a function. Here's how it should look:

```js
const element = event.target
```